protocol/streaming/full_node_streaming_manager_test.go

```diff
@@ -187,7 +187,7 @@ func NewPriceUpdate(
 				MarketId: 1,
 				Price: pricestypes.MarketPrice{
 					Id:       1,
-					Exponent: 1 ^ -6,
+					Exponent: 1_000_000,
```

Comment:
The intention of this original field might've been to set exponent = -6, which represents `Price * 10 ^ Exponent = 10^(-6)`